2.Nomic_nELISA_Analysis/0.Exploratory_Analysis/PBMC/2.cytokine_visualization.ipynb

```diff
@@ -0,0 +1,209 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    from plotnine import (
</pre><p>Is there a specific reason you need to specific all of these imports? It takes up a lot of lines of code. Could you just import plotnine without specifying?</p><p> </p><p>If there is a specific reason, I recommend adding a code comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>cut down on unused imports</p>